### PR TITLE
425 feat add send your idea link at guide tile

### PR DESCRIPTION
**Pull request by Rejfi**
Add "Share your idea" mechanism

### CODE REVIEW

**Comment by kamilfedio** [comment]
I think I am not person to review this so I added @mikolaj-jalocha :D

File: lib/features/guide_view/guide_view.dart [hunk 1]
```diff
@@ -70,13 +71,28 @@ class _GuideViewContent extends ConsumerWidget {
...
+      await launchUrlString(url);
+    }
+  }
 
   @override
   Widget build(BuildContext context) {
-    return WideTileCard(
-      title: context.localize.hi_student,
-      subtitle: context.localize.guide_development_info,
+    return Padding(
```
Comment (mikolaj-jalocha) [inline lib/features/guide_view/guide_view.dart:88]: do we need this padding?

File: lib/features/guide_view/guide_view.dart [hunk 2]
```diff
@@ -70,13 +71,28 @@ class _GuideViewContent extends ConsumerWidget {
...
+      padding: EdgeInsets.zero,
+      child: WideTileCard(
+        title: context.localize.hi_student,
+        subtitle: context.localize.guide_ideas_info,
```
Comment (mikolaj-jalocha) [inline lib/features/guide_view/guide_view.dart:92]: Is some kind of mark (that following text's url) clearly visible?

**Review by mikolaj-jalocha — CHANGES_REQUESTED** [review]
Please see the comments

File: lib/features/guide_view/guide_view.dart [hunk 3]
```diff
@@ -70,13 +71,28 @@ class _GuideViewContent extends ConsumerWidget {
 }
 
 class _GuideInfo extends StatelessWidget {
-  const _GuideInfo();
+  final Uri emailLaunchUri = Uri(
+    scheme: "mailto",
+    path: "kn.solvro@pwr.edu.pl",
```
[the remark this answers is not in the document]
Comment (mikolaj-jalocha) [reply lib/features/guide_view/guide_view.dart:76]: email path

File: lib/features/guide_view/guide_view.dart [hunk 4]
```diff
@@ -70,13 +71,28 @@ class _GuideViewContent extends ConsumerWidget {
...
+    query: "subject=Pomysł na rozwój ToPWR",
```
[the remark this answers is not in the document]
Comment (mikolaj-jalocha) [reply lib/features/guide_view/guide_view.dart:77]: query

**Review by simon-the-shark — APPROVED** [review]
LGTM

**Pull request merged**